src/python/pants/backend/docker/target_types.py

```diff
@@ -312,6 +313,23 @@ def options(self, value_formatter: OptionValueFormatter, **kwargs) -> Iterator[s
             )
 
 
+class DockerBuildOptionFieldListOfMultiValueDictMixin(ListOfDictStringToStringField):
+    """Inherit this mixin class to provide options in the form of `--flag=key1=value1,key2=value2.
+
+    --flag=key3=value3,key4=value4` to `docker build`.
+    """
```

Comment:
docfmt doing a terrible job. suggest breaking out the leading line..

```suggestion
    """Inherit this mixin class to provide multiple key-value options to `docker build`.
    
        --flag=key1=valueA,key2=valueB --flag=key1=valueC,key2=valueD
    """
```